<launch>
    <arg name="pkg_name" default="cubeeye_camera"/>

    <node name="cubeeye_camera" pkg="$(arg pkg_name)" type="cubeeye_camera_node" output="screen">
        <param name="pkg_name" value="$(arg pkg_name)"/>

    </node>
</launch>
